frontend/packages/knative-plugin/integration-tests/utils/install-serverless-operator.ts

```diff
@@ -0,0 +1,107 @@
+import { execSync } from 'child_process';
+import { protractor, by, browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName, retry } from '../../../../integration-tests/protractor.conf';
+import {
+  installServerlessOperator,
+  firstKebabMenu,
+  machineSetsInput,
+  submitCount,
+  servicesValidation,
+  revisionsValidation,
+  machineCount,
+  knativeServingNS,
+  pageSidebar,
+} from '../views/serverless.view';
+
+import {
+  switchPerspective,
+  Perspective,
+  sideHeader,
+} from '../../../dev-console/integration-tests/views/dev-perspective.view';
+import * as sidenavView from '../../../../integration-tests/views/sidenav.view';
+import * as crudView from '../../../../integration-tests/views/crud.view';
+import * as catalogView from '../../../../integration-tests/views/catalog.view';
+import * as catalogPageView from '../../../../integration-tests/views/catalog-page.view';
+import * as operatorView from '../../../operator-lifecycle-manager/integration-tests/views/operator.view';
+import * as operatorHubView from '../../../operator-lifecycle-manager/integration-tests/views/operator-hub.view';
+
+export const installKnOperator = async () => {
+  await crudView.isLoaded();
+  await browser.wait(until.presenceOf(sidenavView.navSectionFor('Operators')));
+  await sidenavView.clickNavLink(['Operators', 'OperatorHub']);
+  await crudView.isLoaded();
+  await catalogView.categoryTabsPresent();
+  await catalogView.categoryTabs.get(0).click();
+  await catalogPageView.catalogTileFor('OpenShift Serverless Operator').click();
+  await browser.wait(until.visibilityOf(installServerlessOperator));
+  await installServerlessOperator.element(by.linkText('Install')).click();
+  await operatorHubView.createSubscriptionFormLoaded();
+  expect(operatorHubView.createSubscriptionFormName.getText()).toEqual(
+    'OpenShift Serverless Operator',
+  );
```

Comment:
you shouldn't be calling expect in a functional utility. This should be part of the test case. Whatever you want to test should be returned from this function

Reply:
I can't remove the expect statements from this function. Please have a look what it actually does and it doesn't make sense to go back to scenario file and then come back to utility just for the expect statements. And for that, I need to split the utility function in multiple functions and it will become a mess.

Reply:
Gaja, Please make it as a rule to keep the expect statements in scenario files